lib/krikri/index_querier.rb

```diff
@@ -0,0 +1,17 @@
+module Krikri
+  # Queries the Search Index
+  class IndexQuerier
```

Comment:
Re:testing, I am not sure what I could test that isn't just re-testing Blacklight functionality, so I would welcome suggestions.